woocommerce/woocommerce
# [CYS on Core] Update image to have a better resolution

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR replaces the `white-black-black-and-white-photograph-monochrome-photography.jpg` image with one with a better resolution.




Closes https://github.com/woocommerce/woocommerce/issues/44610



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure the `WooCommerce Beta Tester` plugin is installed and activated (available on this monorepo).
2. Head over to `/wp-admin/tools.php?page=woocommerce-admin-test-helper` and enable `customize-store` feature flag.
3. Visit the `wp-admin/admin.php?page=wc-admin&path=/customize-store`.
4. Click on `Customize your store` and finish the flow.
5. Check the resolution of the woman in a hat has good resolution and it's not blurry anymore.

| Before | After |
|--------|--------|
| <img width="756" alt="Screenshot 2024-02-14 at 14 27 53" src="https://github.com/woocommerce/woocommerce/assets/186112/7d735daf-5655-4fea-94ca-f94f7c37add7"> | <img width="1880" alt="Screenshot 2024-02-14 at 14 31 00" src="https://github.com/woocommerce/woocommerce/assets/186112/415b217e-fd10-4307-8a05-5e73c1a09bd0"> | 





### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
